gpkit/small_scripts.py

```diff
@@ -73,18 +60,10 @@ def is_sweepvar(sub):
         if sub[0] == "sweep":
             if isinstance(sub[1], Iterable) or hasattr(sub[1], "__call__"):
                 return True
-    except:
+    except:  # pylint: disable=bare-except
```

Comment:
if really necessary, this could use an inline comment justifying, as in "great inline comment" above.